addons/fleet/models/fleet_vehicle.py

```diff
@@ -246,7 +259,7 @@ def action_accept_driver_change(self):
         vehicles._close_driver_history()
 
         for vehicle in self:
-            vehicle.future_driver_id.sudo().write({'plan_to_change_car': False})
+            vehicle.future_driver_id.sudo().write({'plan_to_change_car': False, 'plan_to_change_bike': True})
```

Comment:
```'plan_to_change_bike': True``` ? What is the logic behind this ? 

Shouldn't we set the ``plan_to_change_car`` to ``False`` if the vehicle is a car, and ``plan_to_change_bike`` to ``False`` if the vehicle is a bike ?